util/fibers/fiber_socket_base.cc

```diff
@@ -193,7 +193,7 @@ auto LinuxSocketBase::LocalEndpoint() const -> endpoint_type {
   if (fd_ < 0)
     return endpoint;
 
-  DCHECK_EQ(0, fd_ & IS_UDS);
+  CHECK_EQ(0, fd_ & IS_UDS);
```

Comment:
for sake of avoiding stupid crashes for no reason during codepaths that are not covered by our tests I suggest returning an empty endpoint like with fd_ < 0

Reply:
Sure, did it for RemoteEndpoint() as well. Yet I was planning to review all usages of Endpoint() within Dragonfly as we call them for UDS as well